### ISSUE TITLE
Relation manager table actions disappear when table is reorderable.

### ISSUE THREAD

**Issue by eorenstein1** [filamentphp/filament#3232]
### Package

filament/filament

### Package Version

v2.15.1

### Laravel Version

9.21.6

### PHP Version

8.1.7

### Problem description

After adding reorderable() to a table in a Relation Manager, the table actions disappear.

Without reorderable:
![image](https://user-images.githubusercontent.com/46914709/180906426-45397de5-640c-4980-8011-d5ef036a3fef.png)

With reorderable:
![image](https://user-images.githubusercontent.com/46914709/180906750-5621595c-baa4-40f2-9d63-37e716e05abc.png)

### Expected behavior

The table actions shouldn't disappear.

### Steps to reproduce

Create a relation manager and add reorderable() to the return object in the table function.

**Comment by danharrin** [comment]
Yeah it's deliberate, why would you need the actions when reordering?
